src/network/wire/lib.zig

```diff
@@ -14,6 +14,13 @@ const protocol = @import("../protocol/lib.zig");
 
 const Sha256 = std.crypto.hash.sha2.Sha256;
 
+const Transaction = @import("../../types/transaction.zig");
+const OutPoint = @import("../../types/outpoint.zig");
+const OpCode = @import("../../script/opcodes/constant.zig").Opcode;
+const Hash = @import("../../types/hash.zig");
+const Script = @import("../../types/script.zig");
+const CmpctBlockMessage = @import("../protocol/messages/cmpctblock.zig").CmpctBlockMessage;
```

Comment:
if they are only used for testing move them down. Idealy inside the test scope

Reply:
Okay boss

Reply:
@tdelabro I have moved the imports